choreolib/src/test/java/choreo/trajectory/SampleFlippingTest.java

```diff
@@ -4,14 +4,15 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-import choreo.util.AllianceFlipUtil;
+import choreo.util.ChoreoAllianceFlipUtil;
 import org.junit.jupiter.api.Test;
 
 public class SampleFlippingTest {
   private static final double FIELD_LENGTH_2022 = 16.5811;
   private static final double FIELD_LENGTH_2024 = 16.5811;
   private static final double FIELD_WIDTH_2022 = 8.19912;
-  private static final double FIELD_WIDTH_2024 = 8.19912;
+
+  //   private static final double FIELD_WIDTH_2024 = 8.19912;
```

Comment:
Is this meant to be left commented out or should it be removed?

Reply:
All commented-out code should be removed.

Reply:
I'll delete it